Documentation/architecture.rst

```diff
@@ -258,6 +258,13 @@ Tunnel                   node             64k             Max 32k nodes (IPv4+IP
 IPv4 Fragmentation       node             8k              Max 8k fragmented datagrams in flight simultaneously on the node
 ======================== ================ =============== =====================================================
 
+For some BPF maps, the upper capacity limit can be overridden using command
+line options for ``cilium-agent``. A given capacity can be set using
+``--bpf-ct-global-tcp-max``, ``--bpf-ct-global-any-max``,
+``--bpf-nat-global-max`` and ``--bpf-policy-map-max``. Using the
+``--bpf-map-dynamic-size-ratio`` these upper capacity limits determined based on
```

Comment:
Now `--bpf-fragments-map-max` as well.

Reply:
Will add it, thanks. Do you think it would make sense to dynamically scale the fragments map as well?

Reply:
I've been wondering, too. I think we don't expect too many fragmented packets anyway, so it's probably not worth the trouble at this time. Worst case, users can still use the option to set the size manually. So let's leave it out for now?

Reply:
Agree, let's leave it out for now. Also given that it's default size is fairly small. And if the need arises we can still dynamically size it in a follow-up PR.